### DISCUSSION TITLE
Reticulum, RNode, and Nomad Net at Hamvention in Xenia, OH

### DISCUSSION THREAD

**Discussion by kc1awv** [markqvist/Reticulum#306] [Show and tell]
For those that don't know who I am, I'm Steve KC1AWV and I'm one of the members of the M17 Project, a ham radio community of open source developers and hackers looking to proliferate more open source solutions for the Amateur Radio community at large. One of our biggest projects is the M17 Protocol, an open source voice and data mode for VHF+, using 4FSK modulation and the open source voice codec Codec 2.

This weekend in Xenia, OH, is one of the largest ham radio conventions in the United States called Hamvention. At this event, alongside M17 and FreeDV, I will be showing off Reticulum as well. One of the plans that I have is to try and use FreeDV/FreeDATA as a TNC for using Reticulum over HF bands. I expect to have some good success with this, and hopefully come up with some nice documentation on how it was done. The reason to attempt this, is to provide an option for open source data transmission over longer range, lower bandwidth links instead of more commonly available commercial radio modems such as PACTOR, VARA, and the like. Of course, there's no intention to replace things like Winlink, but certainly could be a great alternative to it, and hopefully inspire more folks to come up with programs and systems for use on Reticulum, in ham radio and beyond.

If you're in the area of Xenia, or happen to be also attending Hamvention, please come see me at booth 1909 in Building 1. It would be great to meet other people interested in Reticulum and see what cool things can be done with it using ham radio!

**Comment by faragher** [comment]
Wish I was within an unplanned day-trip from Xenia! I'll have to look into M17; I started studying for Technician, so maybe it'd be nice to get in touch with some Hams with similar interests.

If you haven't seen it, I did do a proof of concept that LoRa is an encoding method and not encryption: https://github.com/faragher/LoRa-Clear-Broadcast (not to mention it's easily decodable by anyone with publicly available hardware)

Have fun and good hunting!

**Reply by kc1awv** [reply]
Licensing isn't required to participate in the community! As a matter of fact, there's a lot of available resources in the community that can help you on the journey.

As far as the belabored "encryption isn't allowed" argument, I'll reference a thread from a little while back (#70). The important point that @markqvist makes is that the word "encryption" is never mentioned in Part 97. And, an excerpt from a similar discussion on the M17 Discord:

> I like to think of it this way: Is the FCC going to come banging on my door because I'm using Reticulum over LoRa and M17? Most likely not. Will there be angry hams that are going to complain? Probably. Can I prove that my intent isn't to obscure intent? Most definitely.

Anyway, all the relevant info for M17 is at m17project.org. Hope Mark doesn't mind me plugging it a bit here :smile:

**Reply by kc1awv** [reply]
Oh, wait... you were part of that discussion! Ha! :laughing:

**Reply by faragher** [reply]
Thanks! I'll check it out.

Re: Encryption: I used to work in an industry where we lived or died on Federal compliance. I may be overly concerned. 😆

**Reply by markqvist** [reply]
> Hope Mark doesn't mind me plugging it a bit here :smile:

Not at all, I wholeheartedly support the M17 project, so please do. If it wasn't for the fact that all my development time is tied up with Reticulum-related stuff, I'd probably help out with M17 as well :)

**Comment by markqvist** [comment]
Awesome initiative! Wish I could be there in person, but wishing you best of luck with it!

**Comment by zenith777777** [comment]
Hi KC1AWV,

I would be very interested in seeing a Reticulum interface for FreeDATA. Would you mind sharing the documentation on how you did that when it's available?

**Reply by kc1awv** [reply]
It'll probably be done with [freedv-tnc](https://github.com/xssfox/freedv-tnc) at first, since that's the quickest to get a PoC going. Already has the bits and bobs for a KISS interface which should be easy to connect with Reticulum.
